Tests/TestKernel.php

```diff
@@ -141,7 +141,18 @@ protected function configureContainer(ContainerBuilder $container, LoaderInterfa
                 ],
                 'platform_service' => SqlitePlatform::class,
             ],
-            'orm' => null,
+            'orm' => [
+                'auto_mapping' => true,
+                'mappings' => [
+                    'App' => [
+                        'type' => 'annotation',
+                        'dir' => '../docs/resources/entities',
```

Comment:
Perhaps another contributor or maintainer could figure out what the correct doctrine mapping source is in the project?